src/scss/mfe/_forms.scss

```diff
@@ -22,15 +22,15 @@ $background-color_4: #178253e6;
 $background-color_5: #1fad6f;
 $background-color_6: #c32d3ae6;
 $background-color_7: #d64d59;
-$background-color_8: var(--primary);
-$background-color_9: rgba(var(--primary), 0.8); // transparent-ish primary color?
+$background-color_8: var(--primary-500);
+$background-color_9: rgba(var(--primary-500), 0.8); // transparent-ish primary color?
```

Comment:
I can't remember - does this work? We might be able to use `hsla` instead...

I was thinking maybe we can create a new transparent color map specifically for this, but maybe it's beyond the scope of this update.